src/desktop/email.rs

```diff
@@ -177,11 +176,9 @@ impl EmailRequest {
         self
     }
 
-    // TODO Added in version 4 of the interface.
```

Comment:
Please keep this todo bit, as we would like to potentially only send stuff that are supported by the portal (micro-optimization, probably worthless but oh well)